src/Identity/README.md

```diff
@@ -1,7 +1,35 @@
-ASP.NET Core Identity
-=====================
+# Identity
 
-ASP.NET Core Identity is the membership system for building ASP.NET Core web applications, including membership, login, and user data. ASP.NET Core Identity allows you to add login features to your application and makes it easy to customize data about the logged in user.
+ASP.NET Core Identity is the membership system for building ASP.NET Core web applications, including membership, login, and user data. ASP.NET Core Identity allows you to add login features to your application and makes it easy to customize data about the logged in user. You can find additional information in the [ASP.NET Core Documentation](https://docs.microsoft.com/aspnet/core/security/authentication/identity).
```

Comment:
Are we still saying ASP.NET _Core_ web applications?

Reply:
*shrug* only .NET got rebranded, not ASP.NET Core.